src/index.html

```diff
@@ -6,9 +6,34 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo"
+      alt="logo"><img src="./images/logo.png" alt="moyo_logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link"><a href="#" class="is-active">Apple</a></li>
```

Comment:
It will be clearer if you separate the tags on different lines for all your cases.
Currently you don't have actual link addresses and only the `#' character, but imagine when that happens, the length will increase and it will make the markup even more difficult to reading.
```suggestion
          <li class="nav__link">
            <a href="#" class="is-active">Apple</a>
          </li>
```